src/lang/vulnerabilities_es_ES.xml

```diff
@@ -27,11 +27,7 @@ A very visible example was the 2005 hack of the Harvard Business School's applic
  
 Insufficient Data Authorization
 
-Many applications expose underlying data identifiers in a URL. For example, when accessing a medical record on a system one might have a URL such as:
-
-http://example.com/RecordView?id=12345
```

Comment:
Seems the example (and part of previous phrase) was lost during the translation.